common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3029,6 +3029,18 @@
     ],
     "sqlState" : "42846"
   },
+  "UNEXPECTED_VALUE_FOR_LENGTH_IN_SLICE_FUNCTION" : {
```

Comment:
Could you create a sub-class of the error class `INVALID_PARAMETER_VALUE`, and follow formats of other sub-classes like:
```
expects ..., but got ... .
```